File: C#/problem2.cs
using System;

class Problem2
{
	public static void Main()
	{
		int sum = 0;
		for (int i = 1; ; i++)
		{
			if (fib(i) > 4000000)
				break;
			if (fib(i) % 2 == 0)
				sum += fib(i);
		}
		Console.WriteLine(sum);
	}

	public static int fib(int n)
	{
		if (n == 1)
			return 1;
		else if (n == 2)
			return 2;
		else
			return fib(n-1)+fib(n-2);
	}
}

File: C++/Problem2.cpp
#include <stdio.h>

int main() {
    int a = 1, b = 2, sum = 0;

    while (b <= 4000000) {
        if (b % 2 == 0) {
            sum += b;
        }

        int temp = a + b;
        a = b;
        b = temp;
    }

    printf("%d\n", sum);

    return 0;

}
